distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Type_Refinements/In_Memory_Column_Refinements.enso

```diff
@@ -19,7 +19,7 @@ refine_column (column : Column & In_Memory_Column) =
         Value_Type.Decimal _ _ -> _refine_numeric_column column
         _ -> _refine_other_column column
 
-private _refine_numeric_column (column : Column & In_Memory_Column) -> Column & In_Memory_Column & Numeric_Column = case column.length of
+private _refine_numeric_column (column : Column & In_Memory_Column) -> Column & In_Memory_Column & Numeric_Column & Any = case column.length of
```

Comment:
Explanation is [here](https://github.com/enso-org/enso/pull/13593/files#r2228188967)...

Reply:
At the end the [requirements are different](https://github.com/enso-org/enso/issues/13390#issuecomment-3118545211). It is not about `Integer`, but other types.